esp/services/ws_ecl/ws_ecl_service.cpp

```diff
@@ -558,7 +558,16 @@ static void buildReqXml(StringArray& parentTypes, IXmlType* type, StringBuffer&
             else
                 attrval = attr->getSampleValue(s);
             if (attrval)
-                appendXMLAttr(out, attr->queryName(), attrval);
+                appendXMLAttr(out, attr->queryName(), attrval, nullptr, true);
+        }
+        if (flags & REQSF_ROOT)
+        {
+            bool log = reqTree->getPropBool("@log", false);
+            if (log)
+                appendXMLAttr(out, "log", "true", nullptr, true);
+            const char *tracelevel = reqTree->queryProp("@traceLevel");
```

Comment:
Why not getPropInt (as it is above) ?

Reply:
I'll change it.  Was necessary for JSON to get the type formatting right, but would be better here as well.